fixtures/eurolite/led-ip-par.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED IP PAR",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Alex"],
+    "createDate": "2024-04-02",
+    "lastModifyDate": "2024-04-02"
+  },
+  "links": {
+    "manual": [
+      "https://www.steinigke.fr/download/51914172-Instructions-125490-1.000-eurolite-led-ip-par-14x10w-hcl-de_en.pdf"
+    ]
+  },
+  "availableChannels": {
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Amber": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber"
+      }
+    },
+    "UV": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Intensity": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [1, 5],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [6, 10],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [11, 255],
+          "type": "StrobeSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        }
+      ]
+    }
+  },
+  "modes": [
+    {
+      "name": "8",
```

Comment:
Has 3/5/6/12-ch modes, too.

Color presets look very similar (but not identical to) `eurolite/led-tl-3-rgb-uv` and `eurolite/led-tl-4-qcl` -- maybe copy one of those as a starting point.